internal/testplanet/satellite.go

```diff
@@ -115,12 +115,14 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 					AuditReputationBeta0:         0,
 					AuditReputationLambda:        1,
 					AuditReputationWeight:        1,
+					AuditReputationDQ:            0.5,
```

Comment:
I think the default values in the non-test config is 0.6. Is there a reason we're using 0.5 here?

Reply:
I used the defaults that was set yesterday, but those changed later IRCC

I gonna update them to match the current defaults

Reply:
I updated these values to match the current defaults in `pkg/overlay.Config`